src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -551,6 +551,40 @@ Tests:
     - a: "Hello random person: Huey"
     - a: "Hello random person: IonKmglImH"
 
+  - Name: Concat two singletons
+    GivenTemplate:
+      a: {^Concat: {arrays: [["It"], ["worked"]]}}
```

Comment:
Nice! Thanks for adding so many tests to it! Can you also add an example and basic docs to the `Generators.yml` to show this off?

Reply:
Done!